nixos/modules/services/mail/postfix.nix

```diff
@@ -771,18 +767,13 @@ in
         recipient_canonical_classes = [ "envelope_recipient" ];
       }
       // optionalAttrs cfg.enableHeaderChecks { header_checks = [ "regexp:/etc/postfix/header_checks" ]; }
-      // optionalAttrs (cfg.sslCert != "") {
-        smtp_tls_CAfile = cfg.sslCACert;
-        smtp_tls_cert_file = cfg.sslCert;
-        smtp_tls_key_file = cfg.sslKey;
-
-        smtp_use_tls = true;
-
-        smtpd_tls_CAfile = cfg.sslCACert;
-        smtpd_tls_cert_file = cfg.sslCert;
-        smtpd_tls_key_file = cfg.sslKey;
-
-        smtpd_use_tls = true;
+      // optionalAttrs (cfg.clientTlsCAFile != "") {
```

Comment:
I wonder if `clientTlsCAFile` is a good config name since currently there's no client/server differentiation in config values. In contrast `tlsChainFiles` does not specify if the value is used for server or client.

You could argument that in default case the option referes to server. But I'm a little bit unsure about that.

I'd love to hear your opinion about that.

Reply:
I prefer `tlsCAFile`. It's short and corresponds to the upstream name.